### ISSUE TITLE
Accessibility bug: DetailsList - Variable Row Heights command bar

### ISSUE THREAD

**Issue by qazaahir** [microsoft/accessibility-insights-web#1732]
## Describe the bug

The aria allowed attributes aria-orientation-"horizontal" is failing. We don't have access to this variable and cannot make the changes 

**To Reproduce**
If you run 

https://developer.microsoft.com/en-us/fabric#/controls/web/detailslist/variablerowheights

in code pen you will observe the bug 

## Screenshots

![image](https://user-images.githubusercontent.com/57369169/69381034-ba4d9480-0c68-11ea-8781-ec89e79dc2ae.png)

![image](https://user-images.githubusercontent.com/57369169/69381071-cafe0a80-0c68-11ea-9520-2fd1d8047e07.png)

**Comment by JGibson2019** [comment]
@qazaahir this is an issue with OfficeFabric and not with Accessibility Insights for Web. The issue is documented here: https://github.com/OfficeDev/office-ui-fabric-react/issues/11039. Please reach out to the Office Fabric React UI team directly.
